apps/web/screens/Home.tsx

```diff
@@ -34,13 +37,15 @@ interface HomeProps {
   frontpageSlides: GetFrontpageSliderListQuery['getFrontpageSliderList']['items']
   namespace: GetNamespaceQuery['getNamespace']
   news: GetNewsListQuery['getNewsList']['news']
+  getLifeEvents: GetLifeEventsQuery['getLifeEvents']
```

Comment:
I think lifeEvents would be a better variable name for the component, even if the graphql query returns the name getLifeEvents